content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -88,13 +86,13 @@ to read and modify Node objects.
 
 `v1/Node`:
 
-- Get
-- List
-- Create
-- Update
-- Patch
-- Watch
-- Delete
+- get
+- list
```

Comment:
Is it important to use lowercase for the method names?

Reply:
good point, at least this has to be consistent. will revise.

Reply:
Checked, lowercase is correct.
This PR is good to go as is.